Content.Server/Atmos/Piping/Binary/EntitySystems/GasRecyclerSystem.cs

```diff
@@ -0,0 +1,111 @@
+using Content.Server.Atmos.EntitySystems;
+using Content.Shared.Atmos.Piping;
+using Content.Server.Atmos.Piping.Binary.Components;
+using Content.Server.Atmos.Piping.Components;
+using Content.Server.NodeContainer;
+using Content.Server.NodeContainer.Nodes;
+using Content.Shared.Atmos;
+using Content.Shared.Audio;
+using Content.Shared.Examine;
+using JetBrains.Annotations;
+
+namespace Content.Server.Atmos.Piping.Binary.EntitySystems
+{
+    [UsedImplicitly]
+    public sealed class GasReyclerSystem : EntitySystem
+    {
+        [Dependency] private readonly AtmosphereSystem _atmosphereSystem = default!;
+        [Dependency] private readonly SharedAmbientSoundSystem _ambientSoundSystem = default!;
+
+        private const float minTemp = 300 + 273; // 300 C
+        private const float minPressure = 3000;  // 3 MPa
```

Comment:
```suggestion
        private const float MinTemp = 300 + Atmospherics.T0C; // 300 C
        private const float MinPressure = 30 * Atmospherics.OneAtmosphere;  // 3 MPa
```

c# convention + define this in terms of other atmos constants